Tests/Common/Securities/OptionPriceModelTests.cs

```diff
@@ -425,38 +427,129 @@ public void ThrowsIfOptionStyleIsNotSupportedByQLPricingModel(string qlModelName
-        [TestCase(OptionRight.Put, 250, 54.08980489, -0.76656286, 0.00612336, -0.01530234, 0.61233641, -2.07837775)]        // deep OTM
-        public void Greeks(OptionRight optionRight, decimal strike, decimal price, decimal ibDelta, decimal ibGamma, decimal ibTheta, decimal ibVega, decimal ibRho)
+        [TestCase(OptionRight.Call, 448.61, 450, 5.95, 0.50931, 0.02742)]         // ATM
+        [TestCase(OptionRight.Call, 448.84, 470, 0.32, 0.06535, 0.01160)]         // deep OTM
+        [TestCase(OptionRight.Call, 448.85, 430, 21.77, 0.86232, 0.0136)]         // deep ITM
+        [TestCase(OptionRight.Put, 448.77, 450, 5.82, -0.50271, 0.02896)]         // ATM
+        [TestCase(OptionRight.Put, 448.83, 470, 21.16, -1, 0)]                    // deep ITM
+        [TestCase(OptionRight.Put, 449.07, 430, 1.40, -0.13734, 0.01143)]         // deep OTM
+        public void MatchesIBGreeks(OptionRight optionRight, decimal spot, decimal strike, decimal price, decimal ibDelta, decimal ibGamma)
         {
-            const decimal underlyingPrice = 200m;
-            const decimal underlyingVol = 0.25m;
+            const decimal underlyingVol = 0.13m;
             var tz = TimeZones.NewYork;
-            var evaluationDate = new DateTime(2015, 2, 19);
+            var evaluationDate = new DateTime(2023, 8, 10);
             var spy = Symbols.SPY;
-            var optionSymbol = GetOptionSymbol(spy, OptionStyle.American, optionRight, strike);
+            var optionSymbol = GetOptionSymbol(spy, OptionStyle.American, optionRight, strike, new DateTime(2023, 8, 11));
 
             // setting up
-            var equity = GetEquity(spy, underlyingPrice, underlyingVol, tz);
+            var equity = GetEquity(spy, spot, underlyingVol, tz);
             var contract = GetOptionContract(optionSymbol, spy, evaluationDate);
             var option = GetOption(optionSymbol, equity, tz);
             option.SetMarketPrice(new Tick { Value = price });
 
-            // running evaluation
-            var priceModel = OptionPriceModels.BjerksundStensland();
+            // running evaluation for the default model
+            var priceModel = OptionPriceModels.BinomialCoxRossRubinstein();
             var results = priceModel.Evaluate(option, null, contract);
             var greeks = results.Greeks;
 
             // Expect minor error due to interest rate and dividend yield used in IB
-            Assert.AreEqual((double)greeks.Delta, (double)ibDelta, 0.005);
-            Assert.AreEqual((double)greeks.Gamma, (double)ibGamma, 0.005);
-            Assert.AreEqual((double)greeks.Theta / 365.25, (double)ibTheta, 0.005);
-            Assert.AreEqual((double)greeks.Vega, (double)ibVega, 0.005);
-            Assert.AreEqual((double)greeks.Rho, (double)ibRho, 0.005);
+            Assert.AreEqual((double)greeks.Delta, (double)ibDelta, 0.07);
+            Assert.AreEqual((double)greeks.Gamma, (double)ibGamma, 0.07);
```

Comment:
1- Seems to be worse in this case?
2- Shouldn't we still compare `Rho/Vega/etc`, why delete them?
3- Think the accepted delta should be a % from the value itself, so it's relative? Same for the unit test below

Reply:
1. The previous data is generated from a spreadsheet, the current ones are really from IB
2. The Rho/Vega is not available when I fetch these data
3. The greeks themselves are already derivatives/relatives to the underlying (like %), so although they look like comparing in absolute values, they are actually relative error!